hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/store/driver/impl/StateStoreZooKeeperImpl.java

```diff
@@ -63,8 +72,17 @@ public class StateStoreZooKeeperImpl extends StateStoreSerializableImpl {
       RBFConfigKeys.FEDERATION_STORE_PREFIX + "driver.zk.";
   public static final String FEDERATION_STORE_ZK_PARENT_PATH =
       FEDERATION_STORE_ZK_DRIVER_PREFIX + "parent-path";
+  public static final String FEDERATION_STORE_ZK_CLIENT_THREADS_SIZE =
+      FEDERATION_STORE_ZK_DRIVER_PREFIX + "client.size";
+  public static final int FEDERATION_STORE_ZK_CLIENT_THREADS_SIZE_DEFAULT = 10;
+  public static final String FEDERATION_STORE_ZK_CLIENT_CONCURRENT =
+      FEDERATION_STORE_ZK_DRIVER_PREFIX + "client.concurrent";
+  public static final boolean FEDERATION_STORE_ZK_CLIENT_CONCURRENT_DEFAULT = false;
```

Comment:
Can you use one configuration to control whether use this async mode or not? such as the number of thread, default value is -1 or 0, and use can enable this feature by configuring this value with a positive integer.

And we should add detailed explanation of this configuration in hdfs-rbf-default.xml

Reply:
Thank you very much for your help! Just fixed all above problems, please check it again~